tests/Phinx/Migration/Manager/EnvironmentTest.php

```diff
@@ -411,4 +411,51 @@ public function testGettingInputObject()
         $inputObject = $this->environment->getInput();
         $this->assertInstanceOf('\Symfony\Component\Console\Input\InputInterface', $inputObject);
     }
+
+    public function testExecuteMigrationCallsInit()
+    {
+        // stub adapter
+        $adapterStub = $this->getMockBuilder('\Phinx\Db\Adapter\PdoAdapter')
```

Comment:
try to not use the mock in future 
this deprecated in new phpunit versions

Reply:
Err, I'm not sure what you mean and that `getMockBuilder` is used heavily in the PHPUnit 8.5 documentation? What would you prefer I replace this with?

Reply:
mock will deprecated in future
no replacement for it 
but goods for now

Reply:
Could you point to a link for what is being deprecated? I do know that the `setMethods` function is being soft deprecated in 8.x branch, but this is the first I'm hearing that `getMockBuilder` (and mocks in general?) is being deprecated.

Reply:
For now this is fine.

Reply:
yes looks only `setMethods` and `getMock` deprecated now